### ISSUE TITLE
Dropbox uploads corrupting files?

### ISSUE THREAD

**Issue by fasterthanlime** [itchio/itch.io#333]
(Unconfirmed)

Same length, different SHA-1: https://github.com/itchio/itch/issues/404

**Comment by clivecrous** [comment]
After all the testing I've done now, as reported in the issue linked to above by @fasterthanlime , I'm fairly certain something is going wrong with Dropbox uploads.

**Comment by mrhelmut** [comment]
I've already experienced some issues with dropbox doing strange things to executable files (e.g. bash scripts & binaries mostly). Zipping these files resolved this.

**Comment by clivecrous** [comment]
It's not dropbox itself that's the problem. If I download from a dropbox link (within the browser) all's good, but link that same file within itch.io's upload area and it breaks.  The problem is itch.io's Dropbox upload tool.

**Comment by mrhelmut** [comment]
My bad!

**Comment by fasterthanlime** [comment]
> The problem is itch.io's Dropbox upload tool.

At least that's what I too currently suspect, but always keep in mind itch.io tools are used by a lot of users doing a lot of uploads everyday, including dropbox ones, so if it was "broken for everyone", we'd swim in bug reports, not have just the one :)

**Comment by clivecrous** [comment]
@fasterthanlime  indeed, however having downloaded from dropbox, separately from my account, and outside of my dropbox account and having that download work - add to that that presumably Dropbox has even more users doing even more uploads and app syncs ... itch.io is all that's left.
